### DISCUSSION TITLE
Horizontal scroll sample?

### DISCUSSION THREAD

**Discussion by ruimo** [hecrj/iced#883] [Q&A] [answered]
Hi, is there any horizontal scroll sample? I checked all samples in examples directory but I cannot find it.

**Comment by 13r0ck** [comment, marked as answer]
There is no horizontal scroll yet.

**Reply by ruimo** [reply]
Understood. Thank you!

**Reply by lfrancke** [reply]
If others stumble across this as well, this seems to be the related issue: https://github.com/iced-rs/iced/issues/331

**Comment by bungoboingo** [comment]
This will be implemented soon with https://github.com/iced-rs/iced/pull/1550, just need to address some comments!
